tools/cdb2sql/cdb2sql.cpp

```diff
@@ -1699,12 +1735,13 @@ int main(int argc, char *argv[])
         {"cdb2cfg", required_argument, NULL, 'c'},
         {"file", required_argument, NULL, 'f'},
         {"gensql", required_argument, NULL, 'g'},
+        {"delim", required_argument, NULL, 'd'},
```

Comment:
Also add it to [usage_test]()https://github.com/bloomberg/comdb2/blob/master/tools/cdb2sql/cdb2sql.cpp#L127

Reply:
done